lib/ui/screens/login_screen.dart

```diff
@@ -8,146 +8,188 @@ class LoginScreen extends StatefulWidget {
 }
 
 class _LoginScreenState extends State<LoginScreen> {
-  FocusNode _passwordFocusNode;
-
+  FocusNode _passwordFocusNode, _emailFocusNode;
```

Comment:
Please keep these variable declarations seperate

Reply:
Can I know the reason

Reply:
We use separate variable declarations style across whole project in different files. We can't use 2 different styles.